### PR TITLE
feat: #tno-1737 - changed keywords search

### PR DESCRIPTION
**Pull request by kylgarmor**
image

- now uses simple_query_string instead of multi_match
- made textbox much larger
- added some basic help

### CODE REVIEW

File: app/editor/src/features/admin/filters/FilterFormQuery.tsx [hunk 1]
```diff
@@ -105,21 +107,52 @@ export const FilterFormQuery: React.FC = () => {
...
+                <li>
+                  <code>|</code> signifies OR operation
+                </li>
+                <li>
+                  <code>-</code> negates a single token
+                </li>
+                <li>
+                  <code>"</code> wraps a number of tokens to signify a phrase for searching
+                </li>
+                {/* <li><code>*</code> at the end of a term signifies a prefix query</li> */}
```
Comment (jdtoombs) [inline app/editor/src/features/admin/filters/FilterFormQuery.tsx:139]: Probably the case, but just making sure the comments were left intentionally!

Comment (kylgarmor) [reply app/editor/src/features/admin/filters/FilterFormQuery.tsx:139]: yes, I left them hidden as I dont think we need to expose those *yet*

Comment (Fosol) [reply app/editor/src/features/admin/filters/FilterFormQuery.tsx:139]: Does it hurt to expose these now?  This is only visible to admin currently.

File: app/editor/src/features/admin/filters/utils/generateQuery.ts [hunk 2]
```diff
@@ -62,12 +62,23 @@ export const generateQuery = (settings: IFilterSettingsModel, query: any = {}) =
...
-  if (settings.searchIn === 'story')
-    return generateMultiMatch(['summary', 'body'], settings.search);
-  if (settings.searchIn === 'headline') return generateMultiMatch('headline', settings.search);
-  if (settings.searchIn === 'byline') return generateMultiMatch('byline', settings.search);
+  switch (settings.searchIn) {
+    case 'all':
+      // give an arbitrary weight to the headline, so if it's found there
+      // it gets a slightly higher score, as opposed to other fields
+      return generateSimpleQueryString(
+        ['headline^5', 'byline', 'summary', 'body'],
```
Comment (Fosol) [inline app/editor/src/features/admin/filters/utils/generateQuery.ts:70]: This is a neat feature we probably should use more often.

**Review by Fosol — APPROVED** [review]

File: app/editor/src/features/admin/filters/FilterFormQuery.tsx (hunk 1 above)
Comment (kylgarmor) [reply app/editor/src/features/admin/filters/FilterFormQuery.tsx]: fair comment.  I am not sure about how the slop and fuzziness works though TBH.

**Pull request merged**